src/components/FilterTodos/FilterTodos.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import classNames from 'classnames';
+import { FilterType } from '../../types/FilterType';
+
+type Props = {
+  todosCount: number[];
```

Comment:
looks like [tuple](https://www.tutorialsteacher.com/typescript/typescript-tuple) 
```suggestion
  todosCount: [number, number];
```